server/src-lib/Hasura/RQL/Types.hs

```diff
@@ -288,7 +291,11 @@ askFieldInfo m f =
 askCurRole :: (UserInfoM m) => m RoleName
 askCurRole = userRole <$> askUserInfo
 
-successMsg :: EncJSON
-successMsg = "{\"message\":\"success\"}"
+adminOnly :: (UserInfoM m, QErrM m) => m ()
+adminOnly = do
+  curRole <- askCurRole
+  unless (curRole == adminRole) $ throw400 AccessDenied errMsg
+  where
+    errMsg = "restricted access : admin only"
```

Comment:
Is this still needed anymore after the change to move authorization rules into `Query.hs`?